src/cairoWriter/writers/expressionStatementWriter.ts

```diff
@@ -15,12 +15,12 @@ export class ExpressionStatementWriter extends CairoASTNodeWriter {
   writeInner(node: ExpressionStatement, writer: ASTWriter): SrcDesc {
     const documentation = getDocumentation(node.documentation, writer);
     if (
-      node.vExpression instanceof FunctionCall &&
-      node.vExpression.kind !== FunctionCallKind.StructConstructorCall
+      (node.vExpression instanceof FunctionCall &&
+        node.vExpression.kind !== FunctionCallKind.StructConstructorCall) ||
+      node.vExpression instanceof Assignment ||
+      node.vExpression instanceof CairoAssert
     ) {
       return [[documentation, `${writer.write(node.vExpression)};`].join('\n')];
-    } else if (node.vExpression instanceof Assignment || node.vExpression instanceof CairoAssert) {
-      return [[documentation, `${writer.write(node.vExpression)}`].join('\n')];
```

Comment:
Next time please make another PR and set develop as base for cleanup / refactor changes that may be commited on the develop directly. Anything that may be merged to develop should be

Reply:
This PR specifically transform the syntax of the output cairo file, and the changes it propose would break develop. All refractor done here was because of the structure of the new syntax (eg: changes between source unit and contract writters). Is this #962 that I create when working with bools an example of what you want?

Reply:
What I meant is the changes to `expressionStatementWriter` do not change functionality (output) so it could be done on develop. I see it now though that there is a tiny little `;` that is was different between those branches.  So the output has changed and the change may actually be incompatible with cairo 0.x